main/tasks.py

```diff
@@ -273,3 +280,75 @@ def email_monthly_charges_report() -> None:
     email_monthly_charges_report_logic(
         last_month_start.month, last_month_start.year, last_month_name
     )
+
+
+def sync_clockify_time_entries() -> None:
+    """Task to sync time entries from Clockify API to TimeEntry model."""
+    if not settings.CLOCKIFY_API_KEY:
+        logger.warning("Clockify API key not found in environment variables")
+        return
+    days_back = 30
+    api = ClockifyAPI(settings.CLOCKIFY_API_KEY)
+    end_date = timezone.now()
+    start_date = end_date - datetime.timedelta(days=days_back)
+
+    projects = Project.objects.filter(status="Active").exclude(clockify_id="")
+    for project in projects:
+        try:
+            logger.info(f"Processing project ID: {project.clockify_id}")
+            payload = {
+                "dateRangeStart": start_date.strftime("%Y-%m-%dT00:00:00.000Z"),
+                "dateRangeEnd": end_date.strftime("%Y-%m-%dT23:59:59.000Z"),
+                "detailedFilter": {"page": 1, "pageSize": 200},
+                "projects": {"contains": "CONTAINS", "ids": [project.clockify_id]},
+            }
+
+            response = api.get_time_entries(payload)
+            entries = response.get("timeentries", [])
+            if not isinstance(entries, list):
+                entries = []
+
+            for entry in entries:
+                entry_id = entry.get("id") or entry.get("_id")
+                project_id = entry.get("projectId")
+                user_email = entry.get("userEmail")
+                time_interval = entry.get("timeInterval", {})
+                start = time_interval.get("start")
+                end = time_interval.get("end")
+
+                if not (project_id and user_email and start and end):
+                    continue
+
+                try:
+                    project = Project.objects.get(clockify_id=project_id)
+                except Project.DoesNotExist:
+                    logger.info(f"Project with clockify_id {project_id} not found.")
+                    continue
+
+                try:
+                    user = User.objects.get(email=user_email)
+                except User.DoesNotExist:
+                    logger.info(f"User with email {user_email} not found.")
```

Comment:
Make this as warning, as it is important enough.
```suggestion
                    logger.warning(f"User with email {user_email} not found.")
```